src/meltano/cli/__init__.py

```diff
@@ -59,6 +61,28 @@
 
 logger = logging.getLogger(__name__)
 
+troubleshooting_message = """\
+Need help fixing this problem? Visit http://melta.no/ for troubleshooting steps or to
+join our friendly slack community.
+"""
+
+
+def handle_meltano_error(error: MeltanoError):
```

Comment:
This suggestion pairs with the one that imports `NoReturn` above. See https://docs.python.org/3/library/typing.html#typing.NoReturn

```suggestion
def handle_meltano_error(error: MeltanoError) -> NoReturn:
```